tool/python/examples/train_mnist_2.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python
```

Comment:
similarly, we can replace the code 105-115 in train_minst.py with the following loop or add the following code as comments?